buildSrc/src/main/kotlin/criterion/Duration.kt

```diff
@@ -18,17 +18,20 @@ package criterion
 
 import java.lang.System.nanoTime
 
+
 data class Duration(val ns: Double) {
     val ms: Double
         get() = ns / 1e+6
 }
 
+
 inline
```

Comment:
:o: I recall we discussed changing the rule for `inline` to always be in the same line as `fun` and `val` to better align with the standard order of modifiers but I don't remember the outcome.

Reply:
That's correct.
The only enforced rules for now are:
- visibility modifiers order
- visibility modifiers on their own line

We could consider adding another rule, but let's do this after this PR.